WordPress/openverse
# Re-add tags page text

## Fixes



Fast-follow to #4291

## Description


#4442 that added the tags page copy text was merged into #4291, but when I used the GitHub UI to rebase the branch onto `main` before merging it, the merged PR changes disappeared. This PR re-adds the changes from #4442.

Setting the priority to `high` because this affects the strings sent to translators. We can deploy prod again after this PR is merged to speed up the translator strings updates.

## Testing Instructions

Run the app using `just frontend/run dev` (or `just frontend/run i18n:en` and `just frontend/run dev:only` if you don't want to download all locales)
Go to `http://localhost:8443/search?q=cat` and select the second image result.
There should be a "generated tags" section. Click on "Learn more" next to the heading. It should contain correct text (the prod currently has a very short sentence originally used as a stub)

## Checklist


      PRs) or the media properties generator (`./ov just catalog/generate-docs media-props`
      for the catalog or `./ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
